tests/unit_tests/test_wave_06.py

```diff
@@ -261,11 +249,30 @@ def test_swap_best_by_category_no_other_match_is_false():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories
+    assert result == False
+    assert len(tai.inventory) == 3 and len(jesse.inventory) == 3
+    assert tai.inventory == [item_c, item_b, item_a] and jesse.inventory == [item_f, item_e, item_d]
```

Comment:
👍  Nitpick: if I'm testing 2 different data structures (tai's inventory and jesse's inventory) I prefer to have separate asserts just to make the`diff` message from the failed test easier to read. 

This can be applied to the rest of the tests.